gaeguli/pipeline.c

```diff
@@ -586,15 +579,31 @@ _get_source_description (GaeguliPipeline * self)
   return g_string_free (result, FALSE);
 }
 
+static gchar *
+_get_stream_props_description ()
+{
+  GString *stream_props = g_string_new (PIPEWIRE_NODE_STREAM_PROPERTIES_STR);
+
+  g_string_append_printf (stream_props, ",%s=%s-%d",
+      PIPEWIRE_NODE_NAME_STR, GAEGULI_PIPELINE_TAG, getpid ());
```

Comment:
This makes property string like `node.name=gaeguli.pipeline_id-1640394` but what if the client process (one PID) hosts several Gaeguli pipelines? Let's change this to:
```
gaeguli.pipeline_id=1-1640394
```

The number before the dash is a unique identifier of a pipeline within a client process. It can be for example an integer that increments with every `gaeguli_pipeline_new()` call. After the dash is the PID.

Note that I'd prefer we don't modify Pipewire's `node.name` - let's instead create our own custom property (`gaeguli.pipeline_id`).